### PR TITLE
fix: email displayed as organizer eventtype settings

### LINKED ISSUE #14001: [CAL-3268] Event type: Changing the email displayed as the Organizer copy adjustments
There's two things we need to update here.

In the implementation, 

1. there's an extra label on the input that implies we're just sending details to this email address. We are not just sending the email, we are using this person as the organiser. That means most importantly, that we will show that email as the organizer to your booker - thats' the main reason people want this feature. Simply removing that label makes it clearer
2. I know why we added this help text, but we need to make the tooltip actually helpful for the feature as per designs and find another solution for this text. My proposal is we show a toast the first time someone toggles this and never again.
3. We're missing additional help text below the input when this is toggled on

**Current**

**Designs**
CleanShot 2024-03-07 at 09 16 44@2x
CleanShot 2024-03-07 at 09 16 52@2x

Please also pay specific attention to the spacing of the input that appears below when toggled and match it to designs.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3268](https://linear.app/calcom/issue/CAL-3268/event-type-changing-the-email-displayed-as-the-organizer-copy)</sub>

### PR DESCRIPTION
**Pull request by Amit91848**
## What does this PR do?

Fixes #14001 

Accidentally deleted #14042 

Screenshot 2024-03-09 at 1 56 09 PM
Screenshot 2024-03-09 at 1 56 19 PM

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by supalarry — CHANGES_REQUESTED** [review]
Great work Amit ! Thank you for your contributions!

I confirm that the functionality works, but just have a small note about quotes in translation string which I fixed and then  destination calendar input appearing when it should not, which should be fixed. After it is fixed it is good to go! : )

File: apps/web/public/static/locales/en/common.json [hunk 1]
```diff
@@ -804,14 +804,16 @@
...
   "event_name_tooltip": "The name that will appear in calendars",
   "meeting_with_user": "{Event type title} between {Organiser} & {Scheduler}",
   "additional_inputs": "Additional Inputs",
   "additional_input_description": "Require scheduler to input additional inputs prior the booking is confirmed",
   "label": "Label",
   "placeholder": "Placeholder",
-  "display_add_to_calendar_organizer": "Display 'Add to calendar' email as the organizer",
+  "display_add_to_calendar_organizer":"Use 'Add to calendar' email as the organizer",
+  "display_email_as_organizer": "We'll display this email address as the organizer, and send confirmation emails here.",
+  "if_enabled_email_address_as_organizer": "If enabled, we'll display the email address from your 'Add to calendar as organizer and send confirmation emails there",
```
[the remark this answers is not in the document]
Comment (supalarry) [reply apps/web/public/static/locales/en/common.json]: Fixed in [fix: quotation in i18n file](https://github.com/calcom/cal.com/pull/14102/commits/1256a427cf1a3880cafc10cdbb4ad601aabaede0)

File: apps/web/components/eventtype/EventAdvancedTab.tsx [hunk 2]
```diff
@@ -148,96 +148,89 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
...
-                <Label className="text-emphasis mb-0 font-medium">{t("add_to_calendar")}</Label>
-              </div>
-              <Link
-                href="/apps/categories/calendar"
-                target="_blank"
-                className="hover:text-emphasis text-default text-sm">
-                {t("add_another_calendar")}
-              </Link>
+        <div className="flex flex-col space-y-4 lg:flex-row lg:space-x-4 lg:space-y-0">
+          {displayDestinationCalendarSelector && (
```
[the remark this answers is not in the document]
Comment (supalarry) [reply apps/web/components/eventtype/EventAdvancedTab.tsx:152]: To reproduce:

1 in "apps/web/components/eventtype/EventAdvancedTab.tsx" set "const displayDestinationCalendarSelector = false;"

2 login using teampro2 account:
username: teampro2
password: teampro2

3 connect google calendar

4 create an event type and check advanced settings of it

[the remark this answers is not in the document]
Comment (Amit91848) [reply apps/web/components/eventtype/EventAdvancedTab.tsx:152]: Fixed this, accidentally removed some conditions

**Comment by supalarry** [comment]
Hmm, now I don't see the dropdown under the toggle.

What i see:
Screenshot 2024-03-19 at 12 40 29

What is in the design:
image

**Comment by Amit91848** [comment]
Do you have a secondary email added @supalarry ?

**Comment by supalarry** [comment]
Also, I think that we can add to "if_enabled_email_address_as_organizer" explaining that by default cal account e-mail will be used: "If enabled, we'll display the email address from your \"Add to calendar\" as organizer and send confirmation emails there. Otherwise, e-mail used to sign up at Cal is used."

**Comment by Amit91848** [comment]
Works now

**Review by supalarry — APPROVED** [review]
Great work Amit! Appreciate the fast response and thank you for the amazing contribution! 🚀

**Pull request merged**